pt-BR/action_cable_overview.md

```diff
@@ -622,15 +622,15 @@ and unpacked for the data argument arriving as `received`.
 See the [rails/actioncable-examples](https://github.com/rails/actioncable-examples)
 repository for a full example of how to setup Action Cable in a Rails app and adding channels.
 
-## Configuration
+## Configuração
 
-Action Cable has two required configurations: a subscription adapter and allowed request origins.
+O _Action Cable_ tem duas configurações necessárias: um adaptador de assinatura e origens de solicitação permitidas.
```

Comment:
```suggestion
O _Action Cable_ tem duas configurações necessárias: um adaptador de assinatura (_subscription_) e origens de requisição (_request_) permitidas.
```

Acho que aqui a gente pode colocar os termos em inglês também pra ficar mais legal o sentido, o que acha?

Reply:
Uma coisa que notei @rohan-techfreak é que as vezes você coloca nossa sugestão mas não coloca pelo GitHub, se tiver ok pode ir na área de "Files changed" e ir clicando em "Add suggestion to batch" aí no final é só fazer um commit por aqui 😄

Reply:
ok ... implementará desta forma agora em diante

Reply:
é só uma sugestão você pode fazer da forma que achar mais fácil 😄